src/video.h

```diff
@@ -53,7 +53,7 @@ enum LogicUpdateRates {
     LOGIC_15FPS,
     LOGIC_12FPS,
     LOGIC_10FPS,
-    LOGIC_5FPS = 12,
+    LOGIC_5FPS = 12
```

Comment:
I believe the Clang Format gods demand the comma stays.

Reply:
You're right. I fixed what turned out to be a warning from IDO, and not GCC. What's the solution here? Disable IDO warnings, ignore them, or fix them even if it's not cool by Clang Format?